tests/lib/c_lib/common/src/test_sqrt.c

```diff
@@ -13,11 +13,11 @@
 #define local_abs(x) (((x) < 0) ? -(x) : (x))
 
 #ifndef NAN
-#define NAN	(__builtin_nansf(""))
+#define NAN	(__builtin_nans(""))
 #endif
 
 #ifndef NANF
```

Comment:
This code probably shouldn't use NANF -- the C standard (at least C99, C11 and C17) only defines NAN:

> The macro
>
> NAN
>
> is defined if and only if the implementation supports quiet NaNs for the float type. It
> expands to a constant expression of type float representing a quiet NaN.

Similarly, the only defined constant for infinity is `INFINITY`, also of `float` type.

Reply:
ok... well then... I'll guess I'll just remove `NANF` and `INFF` and cast them to floats 🤷 in the sqrtf test

Reply:
done... also, renamed `INF` to `INFINITY`